### PR TITLE
Initial move to `pipenv`

### LINKED ISSUE #156: Move project to pipenv
To modernize the project we want to start using `pipenv` to handle our installation process.

### PR DESCRIPTION
**Pull request by nikola-rados**
This PR moves the requirements handling to `pipenv`. 

Resolves #156.

### CODE REVIEW

File: docker/production/docker-compose.yml [hunk 1]
```diff
@@ -47,7 +52,7 @@ services:
...
         target: /downloads
 
-networks:
-  default:
-    external:
-      name: pcicbr0
+#networks:
+#  default:
+#    external:
+#      name: pcicbr0
```
Comment (nikola-rados) [inline docker/production/docker-compose.yml:58]: Was this meant to be reintroduced at some point? I'm pretty sure we will have issues if we don't include the network here.

Comment (rod-glover) [reply docker/production/docker-compose.yml:58]: Oops.

**Review by rod-glover — APPROVED** [review]
LGTM. LGTY?

**Comment by nikola-rados** [comment]
LGTM

**Pull request merged**